docs/labs/docker-plugin.rst

```diff
@@ -37,6 +37,8 @@ See also the `GitHub repo for this project <https://github.com/ClusterHQ/flocker
     The integration approach allows Docker to control Flocker via the Flocker Dataset API.
     This allows Flocker to be used in conjunction with other ecosystem tools like :ref:`Docker Swarm <labs-swarm>` and :ref:`Docker Compose <labs-compose>`, which is more experimental than ``flocker-deploy`` and the Flocker containers API.
 
+.. _`Docker volumes plugin`: https://github.com/docker/docker/blob/3ee15acaadc3f25c90ea2c4de2e5ab1a12d8e042/docs/extend/plugins_volume.md
```

Comment:
While the previous link can break if Docker choose to move files around, this link is forever stuck at this version, and will not change when significant information changes in the target page.

Since we can see the link break in the (hopefully) unusual case of Docker moving files around, why do we want to change to a more error-prone (success-prone?) link which will always succeed, even if we would have wanted it to change?